setup.cfg

```diff
@@ -34,11 +34,11 @@ classifiers =
 packages = find:
 install_requires =
     ansible-runner >=2rc1,<3
-    jinja2 >=3,<4
+    jinja2
```

Comment:
Can I propose we cap at <4 instead of not having any upper bound at all? It would be really nice to avoid all the hardcoded checks that core has to do. At least if we cap to only support 2.y and 3.y we narrow down how many of those internal checks we need to add. Not having versioned dependencies has bitten us several times in core (in the time I've been there), and particularly on the `ansible-test` side of things, a lot of effort recently has gone into ensuring that everything is capped for what we actually want to support.

I think having no bounds is asking for trouble. We're effectively saying we support things that we have no way of knowing what they will look like

Reply:
can we revisit after 1.0.0?  I think we should have nightly testing to vet any new version of deps that come out, I've reopened https://github.com/ansible/ansible-navigator/issues/453